extensions/2.0/Vendor/EXT_feature_metadata/schema/featureClass.property.schema.json

```diff
@@ -0,0 +1,68 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "title": "Property class",
+  "type": "object",
+  "description": "A property class.",
+  "allOf": [ { "$ref": "glTFProperty.schema.json" } ],
+  "properties": {
+      "name": {
+          "type": "string",
+          "minLength" : 1,
+          "description": "The name of the property, e.g. for display purposes. The string is UTF-8 encoded."
+      },
+      "semantic": {
+          "type": "string",
+          "minLength" : 1,
+          "description": "Identifies a property with a well-known meaning. The string is UTF-8 encoded. Semantics defined by this extension include `NAME` and `ID`. TODO: describe full list of built-in semantics."
+      },
+      "description": {
+          "type": "string",
+          "minLength" : 1,
+          "description": "The description of the property. The string is UTF-8 encoded."
+      },
+      "valueType": {
+          "type": "string",
+          "enum": ["BOOLEAN", "INT8", "UINT8", "INT16", "UINT16", "INT32", "UINT32", "INT64", "UINT64", "FLOAT16", "FLOAT32", "FLOAT64", "STRING", "BINARY"],
```

Comment:
When I run the `gltf-vscode` utility script to set up this schema in my editor, I get a warning:

`enum present without gltf_enumNames`

@lilleyse do you know what this is referring to?

Reply:
The glTF schema has a more verbose way of doing string enums: https://github.com/KhronosGroup/glTF/blob/b5d8a9e3981acb56e75720ae7695e2f5bdf57863/specification/2.0/schema/accessor.schema.json#L74-L102

Maybe we need to do something similar? I'm not sure why it is that way.

Reply:
I prefer the readability of what we have now.